tests/sync_pipfile_test.py

```diff
@@ -54,3 +54,27 @@ def test_sync_dev_pipfile_no_original(tmp_path):
         text = setup_file.read_text()
         assert "pytest~=5.1" in text, text
         assert "requirementslib~=1.5" in text, text
+
+
+@pytest.mark.parametrize(
+    ("source_pipfile_dirname", "update_count"),
+    [("dash_or_underscore_0", 23)],
+)
```

Comment:
When combined with [this suggestion](https://github.com/Madoshakalaka/pipenv-setup/pull/86/files#r752446810):

```suggestion
```

- the `update_count` parameter is not used by this test
- because this test only uses a single dataset, parametrizing the `source_pipfile_dirname` isn't necessary

Reply:
Ah, gotcha, it makes sense to me now.